app/views/passengers/index.haml

```diff
@@ -15,30 +15,29 @@
     = check_box_tag :show_inactive, true, !@filters.include?('active')
   = submit_tag 'Show Passengers'
   = submit_tag 'Print These Filters', name: "print"
-  .checkbox-inline.key_position
-    %table#key
-      %tbody
-        %tr
-          %td.will_expire_soon Will Expire Within 1 Week
-          %td.expired_within_grace_period Expired Within Grace Period
-          %td.inactive Inactive
-          %td.no_note No Note
+  %table#key
+    %tbody
+      %tr
+        %td.will_expire_soon Will Expire Within 1 Week
+        %td.expired_within_grace_period Expired Within Grace Period
+        %td.inactive Inactive
+        %td.no_note No Note
 - if @passengers.present?
-  %table#passengers{class: passengers_table_class}
+  %table#passengers
     %thead
       %tr
         %th Name
         %th Mobility Device
         %th Email
         %th Phone
         - unless @filters.include? 'Active'
-          %th.yesno Active
-        %th.yesno Permanent
-        %th.yesno Expiration Date
-        %th
-        %th
+          %th Active
+        %th Permanent
+        %th Expiration Date
+        %th{data: {orderable: 'false'}}
```

Comment:
I'm on the fence about tossing centering for the icons.  If we want to put it back, though, Bootstrap has `.text-center`.

Reply:
I think I'm almost in favor of having it, just wanted to be minimalist. I'll put it back with bootstrap.